Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -90,7 +84,7 @@
     end
 
     it "includes developer when ENV['HOMEBREW_DEVELOPER'] is set" do
-      ENV["HOMEBREW_DEVELOPER"] = "1"
```

Comment:
I was just wondering if this was a problem yesterday. I haven't seen anything fail because of it though.

Reply:
Oh didn't know this PR was touching this too. I already had #14677 to fix the flaky test, which was probably worthwhile anyway as there were 2-3 PRs failing regularly on it.

Reply:
@Bo98 thanks for the quicker fix 🙇🏻